### ISSUE TITLE
Remove `/blog` from algolia results

### ISSUE THREAD

**Issue by Prinzhorn** [jsreport/website#6]
Or at least move them to the bottom. Right now every time I try to search the docs I end up on blog posts from 2015 or something which are likely entirely irrelevant now.

E.g. the first result is a blog post from **2014**

![Selection_970](https://user-images.githubusercontent.com/679144/131975207-3b89f715-4951-4d89-95ca-b0871b801f55.png)

**Comment by pofider** [comment]
Yes, that makes sense. Thank you for the suggestion. Will try that.
